front-end/cypress/integration/sprint-7-test-119.spec.ts

```diff
@@ -98,7 +98,6 @@ describe('QA Test Script #119 (Sprint 7)', () => {
   it("Steps 4 & 5: Verify 'Candidate Office' dropdown contains House, Senate, and Presidential and then select Presidential", () => {
     cy.get(FieldCandidateOffice).click();
 
-    let OfficeType: string;
     const OfficeTypes: Array<string> = ['House', 'Senate', 'Presidential'];
```

Comment:
Shoot i wish i caught this when i made the other comment.  The variable name should be camelCase: officeTypes and officeType